doc/deprecation_process.md

```diff
@@ -79,6 +79,22 @@ The disclaimer should be added at the very top of the README.md before any other
   - This package has been deprecated and will no longer be maintained after <EOLDate>. This package will only receive security fixes until <EOLDate>. To receive updates on new features and non-security bug fixes, upgrade to the replacement package, [azure-mynewpackage](https://pypi.org/project/azure-mynewpackage/). Refer to the migration guide (https://aka.ms/azsdk/python/migrate/my-new-package) for guidance on upgrading.
   ```
 
+## Remove all links pointing to the package in the main branch
+
+Ensure all links in the samples/README.md and other files that point to the package in the main branch are removed or redirected, as the package will be deleted from main after release.
```

Comment:
Just curious, if we're deleting the package, why do we need to update links on samples/README?

Reply:
So we'll delete the package except for the README, which will point to the latest release tag. And if we don't update the links here, the links in that release tag will be pointing to main, which will return a 404.